### PR TITLE
feat(TDL-31): make edit button black when item is being editable

### PR DESCRIPTION
**Pull request by macopul**

### CODE REVIEW

File: src/ToDoItem/ToDoItem.js [hunk 1]
```diff
@@ -25,47 +25,62 @@ export const ToDoItem = (message) => {
   const closeButton = document.createElement("div");
   closeButton.classList.add("close-button");
 
+  // METHODS
+  
   const checkboxClick = (e) => {
     if (checkbox.checked) {
-      toDoMessage.removeEventListener("click", editClick);
       e.preventDefault();
```
Comment (woopul) [inline src/ToDoItem/ToDoItem.js:32]: add return;

File: src/ToDoItem/ToDoItem.js [hunk 2]
```diff
@@ -25,47 +25,62 @@ export const ToDoItem = (message) => {
...
-    }
-  };
-
-  const editClick = (e) => {
-    if (checkbox.checked == false) {
+    } else {
```
Comment (woopul) [inline src/ToDoItem/ToDoItem.js:33]: remove else - move everyting from inside out

File: src/ToDoItem/ToDoItem.js [hunk 3]
```diff
@@ -25,47 +25,62 @@ export const ToDoItem = (message) => {
...
+      editButton.removeEventListener("mouseout", editButtonHoverOff);
+      toDoMessage.focus();
       e.preventDefault();
+      editButton.style.opacity = 1;
     }
   };
 
-  // METHODS
   const editToDoButton = () => {
     if (checkbox.checked) {
```
Comment (woopul) [inline src/ToDoItem/ToDoItem.js:43]: This if condition does not do anything

**Review by woopul — CHANGES_REQUESTED** [review]

File: src/ToDoItem/ToDoItem.js (hunk 1 above)
Comment (macopul) [reply src/ToDoItem/ToDoItem.js:32]: done

File: src/ToDoItem/ToDoItem.js (hunk 2 above)
Comment (macopul) [reply src/ToDoItem/ToDoItem.js]: done

File: src/ToDoItem/ToDoItem.js (hunk 3 above)
Comment (macopul) [reply src/ToDoItem/ToDoItem.js]: done

**Comment by woopul** [comment]
There is still a bit functionality missing - when you click on edit button it changes to black color, but when you stop editing either on clickOuttside or just by marking task as done, edit button still remains black

https://user-images.githubusercontent.com/21220135/211220485-ffabdd57-f5ea-42a5-9240-56e5425fb21c.mov

**Comment by macopul** [comment]
https://user-images.githubusercontent.com/105385617/211224558-1b2093a2-4cf4-44f3-99ad-7b8983c03c89.mp4

I can not see this problem.
On the newest commit everything works fine in my opinion

**Review by woopul — APPROVED** [review]

**Comment by woopul** [comment]
🎸 looking good

**Pull request merged**